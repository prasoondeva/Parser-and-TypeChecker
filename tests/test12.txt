{
	x : INT;
	z : STRING;
	y : BOOLEAN;
	x = 1;
	z = "a";
	y = < x z;
}