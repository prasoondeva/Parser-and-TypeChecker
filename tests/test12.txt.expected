TYPE MISMATCH 7 C6